### ISSUE TITLE
[Accessibility][Name, Role, Value - Property Pane_Ask a question (Bot Asks)]: Ensures all ARIA attributes have valid values for prompt edit field.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2063]
**User Experience:**
This issue should be resolved for good coding practice as all ARIA attributes have valid values for the edit fields.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
5. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
6. Navigate to property pane appeared at right side of the screen and select 'Bot Asks'.
7. Run Accessibility insight and observe the issue.

**Actual:** 
Edit fields such as prompt, Max turn count, Default value, Allow interruption, Always prompt edit fields doesn't have valid ARIA attributes values.

**Expected:**
Every edit field must have valid aria attributes.

**Note:**
1. Same issue found at "Property Pane_Ask a question (User Asks)".
2. Same issue found at "Property Pane_Ask a question (Other)".
3. Same issue observed at various property panes like Adaptive dialog and other property panes.

**Observation:**
This accessibility issue was found using Accessibility Insights for Web 2.14.1 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**MAS Impacted:** [MAS4.1.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment:**
![MAS4 1 1_Property Pane_Ask a question (User Input)](https://user-images.githubusercontent.com/60913131/75114083-10bbdc00-5679-11ea-93f4-00dbda2f00b5.png)
![MAS4 1 1_Property Pane_Ask a question (Other)](https://user-images.githubusercontent.com/60913131/75114084-11ed0900-5679-11ea-8674-a4d3dae1f583.png)
![MAS4 1 1_Property Pane_Ask a question (Bot Asks)](https://user-images.githubusercontent.com/60913131/75114085-11ed0900-5679-11ea-9402-3d1af51453a1.png)

**Comment by beyackle** [comment]
This appears to be also fixed by a PR I already have up. I'll tag that one here.

**Comment by ashish315** [comment]
@cwhitten As checked on below environment the issue is still repro.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
Browser: Microsoft Edge Canary {Version 83.0.474.0 (Official build) canary (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Comment by ashish315** [comment]
As checked, issue is fixed. So, HCL is closing the issue.
